src/jobs/optimize-image.ts

```diff
@@ -148,6 +149,17 @@ const optimizeImage = async (job: Job) => {
         data: { url: urls },
       });
     } else if (model === "artistAvatar") {
+      artistFavicon = ico.sharpsToIco([sharp(buffer)], "artist_avatar.ico", {
+        sizes: [48],
+        resizeOptions: {},
+      });
+      logger.info("Uploading artist avatar favicon to bucket");
+      const faviconFinalName = `${destinationId}_favicon.ico`;
+      await minioClient.putObject(
+        finalMinioBucket,
+        faviconFinalName,
+        artistFavicon
+      );
```

Comment:
@simonv3 I've tried to store the favicon in Minio as you suggested. Let me know if I got it right 😅 🙏

Reply:
That looks good! If you're running this locally, you can check to see if it ended up there.

Reply:
Had to update the implementation as shown in commit [32d8288](https://github.com/funmusicplace/mirlo/pull/912/commits/32d82880e2cdc1197a71d17a6e367fa6926bb323) but now works (file appears with the name in the artist avatar bucket)

In commit [764fdcb](https://github.com/funmusicplace/mirlo/pull/912/commits/764fdcbcb16879be6d3336a3dab43a0e4fa10c06) I tried to make sure that the favicon file name is unique to each artist avatar.